stumpy/__init__.py

```diff
@@ -174,6 +179,36 @@
         if cd.name == "gpu_aamp_stimp":
             gpu_aamp_stimp.__doc__ = ast.get_docstring(cd)
 
+    # Fix _gpu_searchsorted_left Docs
```

Comment:
Btw, I don't think docstring replacement is necessary here. We only needed to replace docstrings for GPU functions that are at the top-level and really needed for display in our API documentation (at `stumpy.readthedocs.org`). Without it, our documentation on readthedocs doesn't show up correctly. However, since these functions are private, they are not displayed on readthedocs.

Please remove

Reply:
@seanlaw 
Ahhh. I did not know about it! 

I thought the reason for replacing the docstring was to show it to the user in a pop-up mode. (like...in jupyter, if I do `shift+tab`, I can see the docstring) I assume some of editors have this capability. If I remove it, then user will see the docstring of the dummy function. 

Is that okay?

Reply:
I don't know. Given that it's a private function, users wouldn't even know that it exists, right?

Reply:
> Given that it's a private function, users wouldn't even know that it exists, right?

Right :) cool...I will remove it.